prisma/migrations/20220918160012_dev/migration.sql

```diff
@@ -0,0 +1,14 @@
+-- CreateEnum
+CREATE TYPE "ProjectStatus" AS ENUM ('PLANNING', 'IN_PROGRESS', 'SUSPENDED', 'DONE');
+
+-- CreateTable
+CREATE TABLE "Project" (
+    "id" TEXT NOT NULL,
```

Comment:
id is primary key and primary key need to have b-tree index so it's required to set varchar data type for this.

Reply:
PostgreSQL automatically creates an index for each unique constraint and primary key constraint to enforce uniqueness. Thus, it is not necessary to create an index explicitly for primary key columns.

Reply:
but we need VarChar type for b-tree index. (I know it'll created automatically but as the docs that you sent said: you should not use text for a columns which uses B-Tree uindex)